www/include/views/graphs/javascript/centreon-graph.js

```diff
@@ -709,6 +709,10 @@
         legendDiv = jQuery('<div>').addClass('chart-legend')
             .data('curveid', curveId)
             .data('legend', i);
+        legendText = legend.legend;
```

Comment:
Please define the scope of the variable
```suggestion
        let legendText = legend.legend;
```